google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -633,6 +644,12 @@ Builder setCredentials(Credentials credentials) {
       return this;
     }
 
+    @VisibleForTesting
+    Builder setOpenTelemetry(OpenTelemetry openTelemetry) {
```

Comment:
This should be a public setter, so that clients that build `ConnectionOptions` directly (and not only through a connection string) can pass in a specific `OpenTelemetry` instance. This is for example the case for PGAdapter.

Reply:
Updated.